CONTRIBUTING.MD

```diff
@@ -0,0 +1,28 @@
+# Contributing Guidelines
```

Comment:
Should this be CONTRIBUTING.md instead of CONTRIBUTING.MD? Not sure if both works...

Reply:
oops! thanks I'm not sure but I'll lower case the file extension, that was a mistake. 😊